src/Forms/Shared/Samples/GeometryEngine/DensifyAndGeneralize/DensifyAndGeneralize.xaml

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
+             xmlns:local="clr-namespace:ArcGISRuntime.Samples.DensifyAndGeneralize"
+             x:Class="ArcGISRuntime.Samples.DensifyAndGeneralize.DensifyAndGeneralize">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackLayout Grid.Row="0">
+            <Label TextColor="Red"
+                   Text="Adjust the sliders to see the resulting generalized or densified polygon." />
+            <Label FontAttributes="Bold"
+                   Text="Max. Deviation (Generalize)" />
+            <Slider x:Name="DeviationSlider"
+                    Maximum="250" Minimum="1" Value="1" />
+            <Label FontAttributes="Bold"
+                   Text="Max. Segment length (Densify)" />
+            <Slider x:Name="SegmentLengthSlider"
+                    Maximum="500" Minimum="100" Value="100" />
+            <Label x:Name="ResultLabel"
```

Comment:
When the text in this label gets too long it wraps to a second line and pushes the stack down. This makes the UI resize and causes the map view to "twitch" a little when interacting with the slider. I think if you add `FontSize="Small"` to ResultLabel, it won't happen.

Reply:
I retested on all platforms and no longer see this behavior with the fix applied. Thanks!